python_scripts/trees.py

```diff
@@ -626,9 +632,10 @@ def plot_regression_model(X, y, model, extrapolate=False, ax=None):
 _ = plot_regression_model(X_train, y_train, tree)
 
 # %% [markdown]
-# We see that the decision tree model does not have a priori and do not end-up
+# We see that the decision tree model does not have a priori and we do not
```

Comment:
I assume priori = no assumption on data distribution ? 

Maybe make the connection why straight line = no assumption about the data distribution... ?

Reply:
For me also it is not clear what "a priori" means here

Reply:
For me also it is not clear what "a priori" means here.